src/Context/trashContext.js

```diff
@@ -0,0 +1,108 @@
+// import axios from 'axios';
+// import React from 'react'
+// import { useReducer, useContext, createContext} from 'react';
+// import { useNoteContext } from './noteContext';
+
+
+// const TrashContext = createContext();
+// const useTrashContext = ()=> useContext(TrashContext);
+
+// const { noteDispatch } = useNoteContext();
+
+// function trashReducer( state, action ) {
+//     switch(action.type){
+//         case "GET_TRASH_NOTES":
```

Comment:
Suggestion:  Remove this code too if its of no use or do the same as I asked earlier